src/test/resources/toStringBugTest/TestClass2.java

```diff
@@ -2108,37 +2108,7 @@ public Page<Redemption> searchRedemptionsForPay(Long rdmMerchantNo, Long userNo)
     }
 
     private APIErrorCode getRedemptionErrorCode(int status) {
-
-        APIErrorCode apiErrorCode = APIErrorCode.ERR_CATALOGUE_REDEMPTION_FAILED;
-
-        switch( status) {
-
-            case CatalogueRedemptionStatus.CAT_RDM_EXTERNAL_SERVICE_FAILED:
-
-                apiErrorCode =  APIErrorCode.ERR_CATALOGUE_REDEMPTION_FAILED;
-
-            case CatalogueRedemptionStatus.CAT_RDM_BCODE_GENERATION_FAILED:
-
-                apiErrorCode =  APIErrorCode.ERR_CATALOGUE_REDEMPTION_FAILED;
-
-            case CatalogueRedemptionStatus.CAT_RDM_BILL_REBATE_REQUEST_FAILED:
-
-                apiErrorCode =  APIErrorCode.ERR_CATALOGUE_REDEMPTION_FAILED;
-
-            case CatalogueRedemptionStatus.CAT_RDM_VOUCHER_GENERATION_FAILED:
-
-                apiErrorCode =  APIErrorCode.ERR_CATALOGUE_REDEMPTION_FAILED;
-
-            case CatalogueRedemptionStatus.CAT_RDM_REDEEM_FAILED:
-
-                apiErrorCode = APIErrorCode.ERR_CATALOGUE_REDEMPTION_FAILED;
-
-
-
-        }
-
-        return apiErrorCode;
-
```

Comment:
this method always returns APIErrorCode.ERR_CATALOGUE_REDEMPTION_FAILED no matter the given status code is. After this change, we can remove the `int status` parameter too, but maybe some reviewer will notice what `getRedemptionErrorCode` should actually be doing (maybe with a proper switch block).